<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="API.com.project.api.FacultyBioTest" tests="1" time="2.852" ignored="7" failures="1" timestamp="04 Sep 2017 05:54:38 GMT" skipped="0" hostname="IND062" errors="0">
  <testcase name="T1_test_Http_Status_Code_For_The_Programs_API" time="2.852" classname="API.com.project.api.FacultyBioTest">
    <failure type="java.lang.AssertionError" message="Expected and Actual Status code didn&#039;t match expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: Expected and Actual Status code didn't match expected [200] but found [401]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at RestAssureHelper.RestAssuredHelper.assertStatusCodeForResponse(RestAssuredHelper.java:94)
	at API.com.project.api.FacultyBioTest.T1_test_Http_Status_Code_For_The_Programs_API(FacultyBioTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- T1_test_Http_Status_Code_For_The_Programs_API -->
  <testcase name="T4_test_OffSet_And_Max_When_Set_To_Same_Value_Recors_Are_Displayed_While_Using_Pagesize" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T4_test_OffSet_And_Max_When_Set_To_Same_Value_Recors_Are_Displayed_While_Using_Pagesize -->
  <testcase name="T7_test_No_Of_Records_If_Offset_And_Max_Are_Same" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T7_test_No_Of_Records_If_Offset_And_Max_Are_Same -->
  <testcase name="T6_test_Check_Records_In_Range" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T6_test_Check_Records_In_Range -->
  <testcase name="T8_test_No_Of_Records_If_OfSet_IS_SET_TO_0" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T8_test_No_Of_Records_If_OfSet_IS_SET_TO_0 -->
  <testcase name="T2_test_All_Headers_Are_Present_In_The_Response" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T2_test_All_Headers_Are_Present_In_The_Response -->
  <testcase name="T3_test_More_Than_Records_Are_Returned_From_The_Call" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T3_test_More_Than_Records_Are_Returned_From_The_Call -->
  <testcase name="T5_test_Check_A_SpecificRecord" classname="API.com.project.api.FacultyBioTest">
    <skipped/>
  </testcase> <!-- T5_test_Check_A_SpecificRecord -->
</testsuite> <!-- API.com.project.api.FacultyBioTest -->
